flink-formats/flink-csv/pom.xml

```diff
@@ -76,6 +76,12 @@ under the License.
 		</dependency>
 
 		<!-- test dependencies -->
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-base</artifactId>
```

Comment:
What needs this? There is no import in this module to any of the classes, and everything else that might need it (be it a connector or format) should bring their own version.